mne_bids/write.py

```diff
@@ -338,7 +338,28 @@ def _events_json(fname, overwrite=False):
         Whether to overwrite the output file if it exists.
     """
     new_data = {
-        "sample": {"Description": "The event onset time in number of sampling points."},
+        "onset": {
+            "Requirement Level": "REQUIRED",
+            "Data type": "number",
```

Comment:
Neither "Requirement Level" nor "Data type" are needed here, I assume that you got them from the BIDS schema, but please remove them from this PR :-)

IMHO we only need "Description" and "Units" (the latter of which is currently not added in this PR)

Example can be found here: https://github.com/bids-standard/bids-examples/blob/6470461ce3c357080e39a11d7d7b1c4cde9f567b/eeg_matchingpennies/task-matchingpennies_events.json#L2-L8